### ISSUE TITLE
Add 2nd Layer of measure numbers

### ISSUE THREAD

**Issue by Lenbus** [musescore/MuseScore#20829]
### Your idea

Nowadays with the use of tablets, sheet music in `landscape layout` is getting more and more common bringing advantages in terms of less system breaks to read (or more measures per system so to say). 

This comes at the cost of slightly harder navigation in rehearsal sessions as there are fewer measure numbers for orientation.

My way to cope this is to subtly blend in small measure numbers for each and every measure (✅ interval 1, small text ~5 pt, alpha channel = ~100), giving my singers direct measure access without having to count from a system's beginning (which now involves longer passages as to the landscape layout).
![grafik](https://github.com/musescore/MuseScore/assets/12497326/5e45c45a-5cfb-4ac7-9347-57fe6d297bf8)

However, I would really like the first measure number per system to still display as usual, as it...
1.  provides a stronger point of visual orientation 
1.  just does not look good in the beginning of a system e.g. due to interfering with brackets

The same might also come in handy, if we increase to bigger intervals, such as the standard "every 5 bars".

Therefore I would like the two options (`Every System` | `Interval #`) to be checked independently (inclusive OR) with checkboxes instead of exclusively (XOR) by radio buttons. 
![grafik](https://github.com/musescore/MuseScore/assets/12497326/0b1242c7-464d-478a-9985-aacc41d856e2)

### NOTE: 
For this to work perfectly, 
- the two options should rely on different text styles to be formatted independently
- every first measure number per system in the subtle category should be hidden, when "Every System" is checked as well

The intended end result should look something like this:
![grafik](https://github.com/musescore/MuseScore/assets/12497326/14e71d17-106d-4d92-be40-8ab47ea46ffc)

### Problem to be solved

One disadvantage I see is that as systems get longer, measure numbers that are placed only on the beginning of a system tend to be quite sparse, which may be disadvantageous for rehearsing with others due to longer counting passages. 

### Additional context

In short form - this should be easy to implement as everything is already there:
- replace the (XOR) radio buttons in Styles > Measure Numbers by (inclusive OR) checkboxes
- add a new text style for interval measure numbers to be formatted differently from the system beginning measure numbers

Furthermore I think, especially for `Continuous View`, such a display for measure numbers could also be quite beneficial (if rendered in the final export or not).

**Comment by iainhallam** [comment]
Your intended result could also be achieved by allowing start of staff bar numbers to have a different style from in-system numbers. This would need to be implemented by the application for it to work out which numbers should have each style as the location of system breaks may change.

As a workaround, if your layout is fixed (or you don't mind making manual changes when systems break in different places), you can already make the main bar number style be like your small one-per-bar numbers, and define a second style for the start of system numbers, then select all the start of system ones and change the text style to be your new style.

**Comment by andy-lyttle** [comment]
Another workaround might be to insert rehearsal marks at the beginning of each system (if you change the text of the first one from the default "A" to the measure number, any additional rehearsal marks you add should automatically follow the pattern).  Then to work around https://github.com/musescore/MuseScore/issues/20046 you could select the small measure number and press "v" to hide it.

If you don't want the rehearsal marks to be huge and in a box, you can change that style under Style / Test styles / Rehearsal mark.

**Comment by Lenbus** [comment]
Any updates on this? @MarcSabatella, could you be so kind and take this into account for a future update?
It is super easy to implement and quite high in value for most modern sheet music 🙏

**Comment by MarcSabatella** [comment]
FYI, I have no say in what does or doesn't get taken into account. However, I can point out that if you want to add your voice to a request, simply clicking the thumbs up is the recommended method - adding comments of the form "me too" or "when will this be implemented?" is discouraged.
